### PR TITLE
PROSE-175 Revive local environment

### PR DESCRIPTION
**Pull request by asafiniu**
This PR provides a simple way to test/debug ClamAV.

- `Dockerfile`: pull images from `eu.gcr.io`, update `MAINTAINER`
- Add `docker-compose.yml` to enable local testing/debugging using `Dockerfile`
- Unbreak `mvn test` by reinstating missing imports to `ClamAVProxy`
- Add a new test for scanning via `/file` (replaces curl test)
    - Remove `testi.sh`, the curl test
- (git)Ignore maven/vscode project files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,53 +16,32 @@ For more general information, see also [our blog post](http://dev.solita.fi/2015
...
-
-```
-  java -jar clamav-rest-1.0.2.jar --server.port=8765 --clamd.host=myprecious.clamd.serv.er --clamd.port=3310
-```
-
-## Setting up local clamd virtual server
-
-By default clamd is assumed to respond in a local virtual machine. Setting it up is explained in
-[ClamAV client](https://github.com/solita/clamav-java) repository. Or you can use a [clamd Docker image](https://hub.docker.com/r/mkodockx/docker-clamav).
+Alternatively, you can run a docker image for the `clamav-server` and a locally built JAR for `clamav-rest` by running `docker-compose up`.
```
Comment (ehats) [inline README.md:33]: I think this should come first as the preferred option

File: src/test/java/com/tradeshift/clamav/ScanTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import static org.junit.Assert.assertEquals;
+
+import java.nio.charset.StandardCharsets;
+import org.junit.Test;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * These tests assume clamav-rest Docker container is running and responding locally.
```
Comment (ehats) [inline src/test/java/com/tradeshift/clamav/ScanTest.java:12]: it doesn't have to be about docker, right?

Comment (asafiniu) [reply src/test/java/com/tradeshift/clamav/ScanTest.java]: Changed to
```
/**
 * These tests assume clamav-rest is running and responding locally.
```

File: README.md (hunk 1 above)
Comment (asafiniu) [reply README.md]: Made some sense out of the sections on this README, please have a look at [README.md](https://github.com/Tradeshift/clamav-rest/blob/3b5e1fa967032240472314204ad461b4f319c2cc/README.md)

**Review by ehats — APPROVED** [review]

**Pull request merged**